libs/api/domains/air-discount-scheme/src/models/flightLeg.model.ts

```diff
@@ -0,0 +1,30 @@
+import { Field, ObjectType, ID } from '@nestjs/graphql'
+
+import { Flight } from './flight.model'
+
+@ObjectType()
+export class FlightLeg {
+  @Field((_1) => ID)
+  id!: string
+
+  @Field()
+  airline!: string
+
+  @Field({ nullable: true })
+  cooperation!: string
+
+  @Field()
+  financialState!: string
+
+  @Field()
+  travel!: string
+
+  @Field()
+  originalPrice!: number
+
+  @Field()
+  discountPrice!: number
+
+  @Field((_1) => Flight)
+  flight!: Flight
```

Comment:
`_1` => `_`?

Reply:
Not a lot of places where this happens but I guess we can pick `_` as the standard going forward. I don't have a preference, really.